ambry-router/src/test/java/com/github/ambry/router/NonBlockingRouterTest.java

```diff
@@ -1817,6 +1823,28 @@ public void testBlobNotFoundCache() throws Exception {
     }
   }
 
+  /**
+   * Test get blob chunk Ids for composite blob.
+   * @throws Exception
+   */
+  @Test
+  public void testGetBlobChunkIds() throws Exception {
```

Comment:
can you also throw a test case to for simple blob, which should return an empty list.

Reply:
+1

Reply:
Since right now we handle null to empty list mapping in consumer side(getBlobHandler response) to keep consistent with legacy code in this pr, the new test I've added will return null instead of empty list. But I will have a follow up pr to set it as empty list from producer side, so the test will be updated to return an empty list in the future pr.